docs/source/docker/integration/docker_trigger_from_api.rst

```diff
@@ -71,10 +71,9 @@ provided through our Web App or you can use our Python package and build a scrip
       configured to use the data in your AWS S3 bucket. Create such a dataset in 2 steps:
 
       1. `Create a new dataset <https://app.lightly.ai/dataset/create>`_ in Lightly.
-        Make sure that you choose the input type `Images` or `Videos` correctly,
-        depending on the type of files in your cloud storage bucket.
+         Make sure that you choose the input type `Images` or `Videos` correctly, depending on the type of files in your cloud storage bucket.
       2. Edit your dataset, select the storage source as your datasource and fill out the form.
-        In our example we use an S3 bucket.
+         In our example we use an S3 bucket.
```

Comment:
Why 3 spaces indentation?

Reply:
Because the E from edit and the I from In have to start from the same line. There are also 3 spaces indentation here:
https://docutils.sourceforge.io/docs/user/rst/quickref.html